server/routes/CommentRoutes.js

```diff
@@ -1,19 +1,13 @@
 const Comment = require("../controllers/CommentController");
 const router = require("express").Router();
-const { AsyncWrapController } = require("../routes/ErrorHandlingMiddleware");
+const { AsyncWrapController } = require("../middleware/ErrorHandlingMiddleware");
+const checkRole = require("../middleware/CheckRoleMiddleware");
 
 AsyncWrapController(Comment);
 
-// Get a comment for a solution.
 router.get("/:solutionId", Comment.get);
-
-// Create a new comment for a solution in the database.
```

Comment:
why did u delete comments?

Reply:
the route is well defined. I don't see the point of comments there

Reply:
pls keep it